diff --git a/ubuntu-22.04-opencv-full-build.Dockerfile b/ubuntu-22.04-opencv-full-build.Dockerfile
index 8d3ee05..0367175 100644
--- a/ubuntu-22.04-opencv-full-build.Dockerfile
+++ b/ubuntu-22.04-opencv-full-build.Dockerfile
@@ -78,7 +78,7 @@ RUN set -xeu && \
     	-D WITH_CUDA=OFF \
     	-D WITH_CUFFT=OFF \
     	-D WITH_EIGEN=ON \
-    	-D WITH_FFMPEG=ON \
+    	-D WITH_FFMPEG=OFF \
 		-D WITH_FREETYPE=OFF \
     	-D WITH_GDAL=OFF \
     	-D WITH_GDCM=OFF \
@@ -159,7 +159,7 @@ RUN set -xeu && \
 	ls /opt/opencv/include/opencv4 && \
 	ls /usr/include/x86_64-linux-gnu
 
-ENV OPENCV_LINK_LIBS=opencv_videoio,opencv_video,opencv_imgcodecs,opencv_imgproc,opencv_core,libavformat,libavcodec,libavutil,libswscale,liblibjpeg-turbo,liblibpng,liblibopenjp2,ippiw,ippicv,liblibprotobuf,zlib
+ENV OPENCV_LINK_LIBS=opencv_videoio,opencv_video,opencv_imgcodecs,opencv_imgproc,opencv_core,liblibjpeg-turbo,liblibpng,liblibopenjp2,ippiw,ippicv,liblibprotobuf,zlib
 ENV OPENCV_LINK_PATHS=/opt/opencv/lib,/opt/opencv/lib/opencv4/3rdparty,/usr/lib/x86_64-linux-gnu
 ENV OPENCV_INCLUDE_PATHS=/opt/opencv/include,/opt/opencv/include/opencv4
 
@@ -184,6 +184,6 @@ RUN set -xu && \
 	cd /root/oko/backend && \
 	cargo test; exit 0
 
-RUN set -xeu && \
-	cd /root/oko/backend && \
-	cargo test
+# RUN set -xeu && \
+# 	cd /root/oko/backend && \
+# 	cargo test
